community/server/src/main/java/org/neo4j/server/database/LifecycleManagingDatabase.java

```diff
@@ -114,4 +118,18 @@ public boolean isRunning()
     {
         return isRunning;
     }
+
+    private void preLoadCypherCompiler()
+    {
+        // Execute a single Cypher query to pre-load the compiler to make the first user-query snappy
+        //noinspection EmptyTryBlock
+        try (Result ignored = this.graph.execute("MATCH (a:` WEIRD LABEL NO-ONE SHOULD USE. EVER `) RETURN a"))
```

Comment:
We don't pull any results from the lazy query, and when doing read-only queries, Cypher does not create new tokens for labels if they are not already present.